Makefile

```diff
@@ -0,0 +1,44 @@
+TEST_OPTIONS=-m unittest test
+DEBUGGER_PATH=/usr/lib64/python2.7/pdb.py
+MODULE_DIR=modules/$(MODULE).py
+DEBUG_DATA_PATH=test/data/$(DATA).json
+DEBUG_OPTIONS=-m $(MODULE_DIR) -a @$(DEBUG_DATA_PATH) -D $(DEBUGGER_PATH)
```

Comment:
Wouldn't it make more sense to run ```-m pdb $(MODULE_DIR)```?

Reply:
That is an unsupported syntax. I'm not sure how it would work.

```
$ ./.tmp/ansible/hacking/test-module -m pdb modules/katello_content_view.py -a @test/data/content-view.json
Traceback (most recent call last):
  File "./.tmp/ansible/hacking/test-module", line 268, in <module>
    main()
  File "./.tmp/ansible/hacking/test-module", line 249, in main
    (modfile, modname, module_style) = boilerplate_module(options.module_path, options.module_args, interpreters, options.check, options.filename)
  File "./.tmp/ansible/hacking/test-module", line 152, in boilerplate_module
    task_vars=task_vars
  File "/usr/lib/python2.7/site-packages/ansible/executor/module_common.py", line 861, in modify_module
    with open(module_path, 'rb') as f:
IOError: [Errno 2] No such file or directory: 'pdb'
```